bucket/mednaffe.json

```diff
@@ -0,0 +1,53 @@
+{
+    "homepage": "https://github.com/AmatCoder/mednaffe/",
+    "description": "Front-end for Mednafen",
+    "version": "0.9.1",
+    "license": "GPL-3.0-only",
+    "url": "https://github.com/AmatCoder/mednaffe/releases/download/0.9.1/mednaffe-0.9.1-win.zip",
+    "hash": "a19860a17257619315c286c64ae9131b8cd666472bc753dd8cb9a0137c23d895",
+    "bin": "mednaffe.exe",
+    "shortcuts": [
+        [
+            "mednaffe.exe",
+            "Mednaffe"
+        ]
+    ],
+    "installer": {
+        "script": [
+            "$medinst = 0",
+            "if (Test-Path \"$scoopdir\\apps\\mednafen-dev\\current\\mednafen.exe\") {",
+            "   $medinst = 1",
+            "}",
+            "elseif (Test-Path \"$scoopdir\\apps\\mednafen\\current\\mednafen.exe\") {",
+            "   $medinst = 2",
+            "}",
+            "if ((Test-Path \"$scoopdir\\apps\\mednafen\\current\\mednafen.exe\") -and (Test-Path \"$scoopdir\\apps\\mednafen-dev\\current\\mednafen.exe\")) {",
+            "   Write-Host -ForegroundColor Yellow \"Mednafen and Mednafen-dev were found on this system.\"",
+            "   Write-Host -ForegroundColor Yellow \"Choose one to use mednaffe with, 1 (stable) or 2 (dev)\"",
+            "   $medinst = Read-Host -Prompt \"[1 / 2]\"",
+            "}",
+            "switch ( $medinst ) {",
+            "   0 { Write-Warning \"Mednaffe must be installed AFTER mednafen or mednafen-dev. Install either of them, then install mednaffe!\"",
```

Comment:
This message should appear as an error rather than a warning, since the execution is aborted just after printing it.

Reply:
Ok, I've changed it to an error and moved it to `pre-install`. It will also uninstall mednaffe to prevent scoop from saying the install failed in `scoop list`